app/services/DynamoChannelTests.scala

```diff
@@ -65,6 +83,39 @@ class DynamoChannelTests(stage: String, client: DynamoDbClient) extends StrictLo
       DynamoPutError(error)
     )
 
+  private def put(putRequest: PutItemRequest): ZIO[ZEnv, DynamoPutError, Unit] =
+    effectBlocking {
+      val result = client.putItem(putRequest)
+      logger.info(s"PutItemResponse: $result")
+      ()
+    }.mapError(error =>
+      DynamoPutError(error)
+    )
```

Comment:
minor really but you can reduce this I think
```suggestion
    }.mapError(DynamoPutError)
```